src/account-type-utils.js

```diff
@@ -60,7 +60,7 @@ const getAccountType = email => {
  *
  */
 const getEmailDomain = email => {
-  return email.includes('@') && email.includes('.')
+  return _.includes(email, '@') || _.includes(email, '.')
```

Comment:
To avoid further issues, if might actually be easier to just check to make sure that email was specified and return unknown in that case, no?

Reply:
Changed